packages/cli/src/user-management/email/user-management-mailer.ts

```diff
@@ -50,30 +52,22 @@ export class UserManagementMailer {
 		if (!this.mailer) return { emailSent: false };
 
 		const template = await this.getTemplate('invite');
-		const result = await this.mailer.sendMail({
+		return await this.mailer.sendMail({
 			emailRecipients: inviteEmailData.email,
 			subject: 'You have been invited to n8n',
-			body: template(inviteEmailData),
+			body: template({ ...inviteEmailData, ...this.getBasePayload() }),
```

Comment:
Just to be safe, I'd destructure the base payload first and then the template-specific args.